src/cairo_interaction.rs

```diff
@@ -25,16 +25,21 @@ impl GdkSurfaceExt for Surface {
 }
 
 pub trait GdkPixbufExt {
-    fn create_surface<W: IsA<Window>>(&self, scale: i32, for_window: &W) -> Option<Surface>;
+    fn create_surface<W: IsA<Window>>(&self, scale: i32, for_window: Option<&W>)
```

Comment:
This changes API and should not be in the bugfix release

CC @EPashkin @GuillaumeGomez

Reply:
My mistake,  I will add revert tomorrow morning

Reply:
Don't worry, I'm on it.